.golangci.yml

```diff
@@ -104,6 +106,7 @@ linters:
 
   # don't enable:
   # - asciicheck
+  # - exhaustive
```

Comment:
why not enable it?

Reply:
Sounds good, I can enable it.

I did not enable it originally because I was unsure if users would find the default behavior of the analyzer to be too strict. (The default behavior of the analyzer is to point out missing enum members in the switch even if a `default` case is present.)